browser/resources/brave_new_tab_page_refresh/components/widgets/news_widget.tsx

```diff
@@ -0,0 +1,130 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import Button from '@brave/leo/react/button'
+
+import * as mojom from 'gen/brave/components/brave_news/common/brave_news.mojom.m.js'
+import { useBraveNews } from '../../../../../components/brave_news/browser/resources/shared/Context'
+import { getTranslatedChannelName } from '../../../../../components/brave_news/browser/resources/shared/channel'
+import { channelIcons } from '../../../../../components/brave_news/browser/resources/shared/Icons'
+
+import { getString } from '../../lib/strings'
+import { SafeImage } from '../common/safe_image'
+import { Link } from '../common/link'
+
+import { style } from './news_widget.style'
+
+export function NewsWidget() {
+  const braveNews = useBraveNews()
+  return (
+    <div data-css-scope={style.scope}>
+      <div className='title'>
+        {getString('newsWidgetTitle')}
+      </div>
+      {braveNews.isOptInPrefEnabled ? <PeekItem /> : <OptIn />}
+    </div>
+  )
+}
+
+function PeekItem() {
+  const braveNews = useBraveNews()
+  const feedItems = braveNews.feedV2?.items
+
+  const peekItem = React.useMemo(() => {
+    return feedItems ? getPeekItem(feedItems) : null
+  }, [feedItems])
+
+  if (!peekItem) {
+    return (
+      <div className='peek loading'>
+        <div className='img skeleton' />
+        <div className='text'>
+          <div className='meta skeleton' />
+          <div className='item-title skeleton' />
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <Link url={peekItem.url.url} className='peek'>
```

Comment:
I think clicking the peek item shouldn't open the article but scroll you down into News - its a bit annoying but otherwise there's no way to know you can scroll down the page.

Reply:
As a user, I'd expect clicking on the news widget to take me to the news site, but I see your point. Perhaps we should run this by design?

cc @kleantzogu

Reply:
We had a similar discussion for the old news stuff on the current NTP - I kind of agree about opening the article being expected, but it is very helpful for news discovery